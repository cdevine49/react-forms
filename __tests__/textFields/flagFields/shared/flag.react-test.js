import React from 'react';
import Flag from '../../../../lib/components/textFields/flagFields/shared/flag';
import renderer from 'react-test-renderer';

describe("Country", () => {
  test("Without Props", () => {
    const tree = renderer.create(
      <Flag />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  describe("With Props", () => {
    test('offset', () => {
      const tree = renderer.create(
        <Flag offset="-44px" />
      ).toJSON();

      expect(tree).toMatchSnapshot();
    });

    test('className', () => {
      const tree = renderer.create(
        <Flag className="test" />
      ).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});