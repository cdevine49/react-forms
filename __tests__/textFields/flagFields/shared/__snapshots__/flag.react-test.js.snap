// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country With Props className 1`] = `
<div
  className="flag test"
  style={Object {}}
/>
`;

exports[`Country With Props offset 1`] = `
<div
  className="flag"
  style={
    Object {
      "backgroundImage": "url(/images/flags.png)",
      "backgroundPositionX": "0px",
      "backgroundPositionY": "-44px",
    }
  }
/>
`;

exports[`Country Without Props 1`] = `
<div
  className="flag"
  style={Object {}}
/>
`;
